setupJest.js

```diff
@@ -4,7 +4,7 @@
 const { configure } = require('@cloudscape-design/browser-test-tools/use-browser');
 const { devServerPort } = require('./scripts/config');
 
-jest.setTimeout(60 * 1000);
+jest.setTimeout(90 * 1000);
```

Comment:
Just double checking, is this needed?

Reply:
See the original CR-97008919 for the context